src/Microsoft.OData.ModelBuilder/Conventions/Attributes/NullableAttributeExtensions.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System.Linq;
+
+namespace Microsoft.OData.ModelBuilder.Conventions.Attributes
+{
+    /// <summary>
+    /// Extensions for properties that have the NullableAttribute.
+    /// </summary>
+    /// <remarks>This MUST come after RequiredAttribute</remarks>
+    internal static class NullableAttributeExtensions
```

Comment:
>NullableAttributeExtensions [](start = 26, length = 27)

maybe rename the class name as "PropertyConfigurationNullableAttributeExtensions"?

Reply:
Fixed. Renamed to PropertyConfigurationExtensions. The extensions can be considered generally usable so narrowing the name to "NullableAttribute" is not needed.